packages/cli/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "@stylexjs/cli",
+  "version": "0.5.1",
+  "description": "A cli to compile a folder with StyleX",
+  "main": "./src/index.js",
```

Making this change will fix the problem.

```suggestion
  "main": "./lib/index.js",
```